sites/svelte.dev/src/routes/+layout.svelte

```diff
@@ -17,13 +19,13 @@
 
 <div style:display={$page.url.pathname !== '/docs' ? 'contents' : 'none'}>
 	<Shell nav_visible={$page.url.pathname !== '/repl/embed'}>
-		<Nav slot="top-nav">
+		<Nav slot="top-nav" title={data.nav_title} links={data.nav_links}>
 			<svelte:fragment slot="home-large">
 				<strong>svelte</strong>.dev
 			</svelte:fragment>
 
 			<svelte:fragment slot="home-small">
-				<!-- <strong>svelte</strong> -->
+				<strong>svelte</strong>
```

Comment:
I don't think we need this for the svelte site. I left that example as reference for kit and learn